test/ServerTest/RequestAPI.test.js

```diff
@@ -0,0 +1,23 @@
+const request = require('supertest')('http://localhost:3002');
+
+xdescribe ('A suite for server request', () => {
```

Comment:
You don't test this section? just testing one sample?

Reply:
Ooh yeah, this one we test. I have just removed x from "describe" in the new commit. The test is passing still. I was skipping it when I was developing client side tests.